code/__DEFINES/preferences_defines.dm

```diff
@@ -106,6 +106,11 @@
 
 #define EXP_DEPT_TYPE_LIST		list(EXP_TYPE_SUPPLY, EXP_TYPE_SERVICE, EXP_TYPE_MEDICAL, EXP_TYPE_ENGINEERING, EXP_TYPE_SCIENCE, EXP_TYPE_SECURITY, EXP_TYPE_COMMAND, EXP_TYPE_SILICON, EXP_TYPE_SPECIAL, EXP_TYPE_GHOST)
 
+#define GLOW_HIGH    0
+#define GLOW_MED     1 //default.
+#define GLOW_LOW     2
+#define GLOW_DISABLE 3 //this option must be the highest number
```

Comment:
But why? As far as I can see, the code doesn't compare values with anything but equality.